src/kuzya.h

```diff
@@ -173,6 +175,8 @@ private slots:
         QFile *file;
         QString language; //curren programing language
         QsciScintilla* textEditor;
+        QTabWidget *tabWidget;
```

Comment:
It would be more clearly if you called this pointer outputTabWidget. And, don't forget about 'm' prefix in class members.